src/cmap/commands.ts

```diff
@@ -876,4 +876,15 @@ export class BinMsg {
 
     this.parsed = true;
   }
+
+  private _parseBsonSerializationOptions({ enableUtf8Validation }: BSONSerializeOptions): {
+    utf8: { writeErrors: false } | false;
+  } {
+    // eslint-disable-next-line no-restricted-syntax
+    if (enableUtf8Validation === undefined || enableUtf8Validation) {
```

Comment:
I _think_ this should never be undefined, because I set the default value to `true` in both `connection_string.ts` and `resolveBSONOptions`.  But the interface does allow for an undefined, so I decided to handle it.  Checking for `undefined` results in a linting error so I have to use an `eslint-disable`